### PR TITLE
Fixed bug for creating/editing categories as admin

### PR DESCRIPTION
**Pull request by jennaplusplus**
Added 2 cucumber scenarios and 1 rspec test along with fixing admin categories controller bug. Admin users can now successfully edit existing categories and create new categories.

### CODE REVIEW

File: spec/controllers/admin/categories_controller_spec.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@
     request.session = { :user => henri.id }
   end
 
+  it "test_new" do
```
Comment (kariabancroft) [inline spec/controllers/admin/categories_controller_spec.rb:14]: Do you have an RSpec test for the edit or is this pre-existing?

Comment (jennaplusplus) [inline spec/controllers/admin/categories_controller_spec.rb:14]: I believe that editing is covered in the pre-existing `describe "test_edit"` block lower down.

File: features/create_category.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    And I am logged into the admin panel
+
+  Scenario: Successfully create categories
+    Given I am on the admin content page
+    When I follow "Categories"
+    Then I should see "Categories"
+    When I fill in "category_name" with "Something"
+    And I fill in "category_description" with "Lorem Ipsum"
+    And I press "Save"
+    Then I should see "Category was successfully saved."
```
Comment (kariabancroft) [inline features/create_category.feature:17]: You may also want to add something here that would ensure your new category was displayed in the list

File: features/edit_category.feature [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  Scenario: Successfully edit categories
+    Given I am on the admin content page
+    When I follow "Categories"
+    Then I should see "Categories"
+    When I follow "Edit"
+    And I fill in "category_name" with "Something else"
+    And I fill in "category_description" with "Lorem Ipsum again"
+    And I press "Save"
+    Then I should see "Category was successfully saved."
```
Comment (kariabancroft) [inline features/edit_category.feature:18]: Same comment re: updated category

File: spec/controllers/admin/categories_controller_spec.rb (hunk 1 above)
Comment (kariabancroft) [inline spec/controllers/admin/categories_controller_spec.rb:14]: Ok, sounds good!

File: features/create_category.feature (hunk 2 above)
Comment (jennaplusplus) [inline features/create_category.feature:17]: Added this in new commit

File: features/edit_category.feature (hunk 3 above)
Comment (jennaplusplus) [inline features/edit_category.feature:18]: Added this in new commit

**Comment by kariabancroft** [comment]
Looks good - ready to close!

**Pull request closed without merging**